src/components/Table/DataTable/compenents/el-data-table/el-data-table.vue

```diff
@@ -1033,7 +1036,7 @@ export default {
 
           // 开启persistSelection时，需要同步selected状态到el-table中
           this.$nextTick(() => {
-            this.selectStrategy.updateElTableSelection()
+            this.selectStrategy?.updateElTableSelection()
           })
         })
         .catch(err => {
```

Comment:
There is no specific issue mentioned from the provided code that seems to require an answer in English.

However, if you have questions about how these changes should be applied, feel free to ask!